bcda/auth/client/ssas.go

```diff
@@ -378,3 +378,33 @@ func (c *SSASClient) setAuthHeader(req *http.Request) error {
 	req.Header.Add("Accept", "application/json")
 	return nil
 }
+
+// GetVersion Gets the version of the SSAS client
+func (c *SSASClient) GetVersion() (string, error) {
+	req, err := http.NewRequest("GET", fmt.Sprintf("%s/_version", c.baseURL), nil)
+	if err != nil {
+		return "", err
+	}
+
+	if err := c.setAuthHeader(req); err != nil {
+		return "", err
+	}
+	resp, err := c.Do(req)
+	if err != nil {
+		return "", err
+	}
+	defer resp.Body.Close()
+
+	ssasLogger.Info(resp.Body)
+	type SSASVersion struct {
```

Comment:
Nit - can we switch this to be lowercase to make in explicit that this field is not exported/public?